### PR TITLE
Added animations

### LINKED ISSUE #1606: Add animation on opening new Activity
Currently, simple open and close system is working , I would like to add slide type animations to make the app attractive

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fix #1606 

Changes: Added right to left and left to right animations for Camera, Gallery and Accounts.

### CODE REVIEW

**Comment by codedsun** [comment]
@batbrain7 Is the app working smooth as there are already many animations present?
And please attach some screenshots

@mayank8318 @nikit19 Your views?

**Comment by batbrain7** [comment]
@codedsun I have added animations only on the part where we switch from gallery to camera to accounts.
And here is a gif showing animations 
ezgif com-video-to-gif


Please suggest changes if any :)

File: app/src/main/res/anim/left_to_right.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate android:fromXDelta="-60%" android:toXDelta="0%"
+        android:fromYDelta="0%" android:toYDelta="0%"
+        android:duration="150"/>
+</set>
```
Comment (samagra14) [inline app/src/main/res/anim/left_to_right.xml]: POSIX standards indicate that EOF should preferably be a new line. Please look into it.

File: app/src/main/res/anim/right_to_left.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate android:fromXDelta="-60%" android:toXDelta="0%"
+        android:fromYDelta="0%" android:toYDelta="0%"
+        android:duration="150"/>
+</set>
```
Comment (samagra14) [inline app/src/main/res/anim/right_to_left.xml]: POSIX standards indicate that EOF should preferably be a new line. Please look into it.

**Comment by batbrain7** [comment]
@mohitmanuja @anantprsd5 @isuruAb please review

**Comment by batbrain7** [comment]
@anantprsd5 made the changes

**Comment by anantprsd5** [comment]
Please squash your commits.

**Comment by batbrain7** [comment]
@anantprsd5 the PR is done

**Comment by anantprsd5** [comment]
@batbrain7 Why the binary files are added in this PR. Please see if the changes made are appropriate before sending the PR. Thanks

**Comment by batbrain7** [comment]
@anantprsd5 I'll remember it from the next time, the binary files are not present now.

**Comment by batbrain7** [comment]
@isuruAb @anantprsd5 review this PR now

**Pull request merged**